PowerShellTools/ServiceManagement/PowershellHostProcessHelper.cs

```diff
@@ -13,6 +14,18 @@ namespace PowerShellTools.ServiceManagement
     /// </summary>
     internal static class PowershellHostProcessHelper
     {
+        [DllImport("user32.dll", SetLastError = true)]
+        private static extern bool SetWindowPos(IntPtr hWnd, IntPtr hWndInsertAfter, int X, int Y, int W, int H, uint uFlags);
+
+        [DllImport("user32.dll")]
+        private static extern bool ShowWindow(IntPtr hWnd, int nCmdShow); 
+
+        private static readonly IntPtr HWND_TOPMOST = new IntPtr(-1);
+        private const UInt32 SWP_NOSIZE = 0x0001;
+        private const UInt32 SWP_NOMOVE = 0x0002;
+        private const UInt32 TOPMOST_FLAGS = SWP_NOMOVE | SWP_NOSIZE;

```

Comment:
Do we also want SWP_NOACTIVATE, or no?  [Does anything else lose focus/activation, even though we're hidden?]